StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift

```diff
@@ -34,7 +34,7 @@ public extension PaymentSheet {
             _ intentCreationCallback: @escaping ((Result<String, Error>) -> Void)
         ) -> Void
 
-        public typealias CVCRecollectionHandler = () -> Bool
+        public typealias CVCRecollectionEnabledCallback = () -> Bool
```

Comment:
Sorry @wooj I approved, but realized - can we put this under `_spi` too?  And also add a docstring

Reply:
oops, yes. this needs to be under _spi!